test/regressions/index.js

```diff
@@ -78,7 +78,7 @@ const blacklist = [
   'docs-components-popover/AnchorPlayground.png', // Redux isolation
   'docs-components-popover/MouseOverPopover.png', // Needs interaction
   'docs-components-popover/PopoverPopupState.png', // Needs interaction
-  'docs-components-popover/SimplePopover.png', // Needs interaction
+  'docs-components-popover/BasicPopover.png', // Needs interaction
```

Comment:
What we would need 21e2d5394ae82ba062b95cd312191543de7e2e6b to have detected in the PR